vaxrank/report.py

```diff
@@ -71,8 +71,8 @@ def ascii_report_from_ranked_vaccine_peptides(
         for j, vaccine_peptide in enumerate(vaccine_peptides):
             mutant_protein_fragment = vaccine_peptide.mutant_protein_fragment
             amino_acids = mutant_protein_fragment.amino_acids
-            mutation_start = mutant_protein_fragment.mutant_amino_acid_start_offset
```

Comment:
Perfect; this looks good to me! Is there a comment you could stick there explaining why we need to add 1?